example/src/react-barong/index.js

```diff
@@ -41,22 +44,27 @@ var InputError = function (_a) {
 };
 
 var BarongLoginForm = function (_a) {
-    var host = _a.host, redirection = _a.redirection, forgotPasswordUrl = _a.forgotPasswordUrl;
+    var host = _a.host, redirection = _a.redirection, testMode = _a.testMode, forgotPasswordUrl = _a.forgotPasswordUrl;
     var _b = useForm(), register = _b.register, handleSubmit = _b.handleSubmit, errors = _b.errors;
     var onSubmit = useCallback(function (data) {
-        ApiUtil.login(host, data)
-            .then(function (response) {
-            if (response.status === 200) {
-                window.location.replace(redirection);
-            }
-            else {
-                window.console.log(response);
-            }
-        })
-            .catch(function (err) {
-            window.console.error(err);
-        });
-    }, [host, redirection]);
+        if (testMode === true) {
+            window.location.replace(redirection);
+        }
+        else {
+            BarongApiUtil.login(host, data)
+                .then(function (response) {
+                if (response.status === 200) {
+                    window.location.replace(redirection);
+                }
+                else {
+                    window.console.error(response);
+                }
+            })
+                .catch(function (err) {
```

Comment:
fix the indentation

Reply:
this is the example project. The folder "react-barong" contains here the generated js code.